src/random.cpp

```diff
@@ -714,3 +715,10 @@ void RandomInit()
 
     ReportHardwareRand();
 }
+
+std::chrono::microseconds GetExponentialRand(std::chrono::microseconds now, std::chrono::seconds average_interval)
+{
+    double unscaled = -std::log1p(GetRand(1ULL << 48) * -0.0000000000000035527136788 /* -1/2^48 */);
```

Comment:
Is 1ULL guaranteed to be of the same width as uint64_t{1}? Even if they are, maybe still replace it while touching this line?

Also, the function could be moved closer to GetRand*Duration, to bundle related calls?

Reply:
I wasn't able to find systems where it wouldn't be the same width, but I don't think there is a guarantee. Changed it to  `uint64_t{1}`.
Also moved the function up in `random.h` as suggested.